.github/workflows/ci.yml

```diff
@@ -157,6 +157,16 @@ jobs:
         env:
           DONT_RUN_VCPKG: true
 
+      - name: Run unit tests
+        run: cd build && ctest -VV
+        env:
+          DPP_UNIT_TEST_TOKEN: ${{secrets.DPP_UNIT_TEST_TOKEN}}
```

Comment:
what if the mac unit test runs at the same time as the linux one? their messages will conflict and both will fail, because this is how discord api works, both will see each others output. One of them (this one?) should be offline only unit tests.

Reply:
Oh right, yeah Mac can do the offline test then :)